src/chimera/instruments/fakefocuser.py

```diff
@@ -84,3 +95,8 @@ def _setPosition(self, n):
     def _inRange(self, n):
         min_pos, max_pos = self.getRange()
         return (min_pos <= n <= max_pos)
+
+    def _checkAxis(self, axis):
+        if axis != FocuserAxis.Z:
+            print axis != FocuserAxis.Z
+            raise InvalidFocusPositionException("Cannot move %s axis."%axis)
```

Comment:
I think that this function you will use on all focusers. Can you generalize it and move to the base instrument like done on camera here: https://github.com/astroufsc/chimera/blob/eca91491a3cc7712a7e00cf6d92bc18793775621/src/chimera/instruments/camera.py#L149 ?

Could be something like:
```python
def _checkAxis(self, axis):
    if axis not in ListOfAxisThatICanControl:
        raise Caca("caca")
    return True
```

and remove the print ;-)